carmin.yaml

```diff
@@ -246,7 +245,7 @@ paths:
         default:
           $ref: '#/responses/GenericError'
   /executions/{executionIdentifier}/delete:
-    put:
+    delete:
```

Comment:
There's a risk with a DELETE method with a body : #37 
I would be OK to transfer the "deleteFiles" parameter from the body to a query parameter

Reply:
shall we keep PUT for all delete operations then? I noticed that your deletePath in VIP uses DELETE.

Reply:
I prefer to use DELETE everywhere.